tests/api2/test_root_session_alert.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from middlewared.test.integration.utils.client import client, truenas_server
+from middlewared.test.integration.utils import call
+
+
+def get_session_alert(call_fn):
+    alerts = call_fn('alert.list')
+    alert_msg = None
+
+    for alert in alerts:
+        if alert['klass'] == 'AdminSessionActive':
+            alert_msg = alert['formatted']
+            break
+
+    assert alert_msg is not None, str(alerts)
+    return alert_msg
+
+
+def check_session_alert(call_fn):
```

Comment:
What is call_fn?

Reply:
`call` or `c.call` e.g. client call methods.

Reply:
derp, I haven't had coffee yet